lib/pmdtester/cmd.rb

```diff
@@ -3,14 +3,13 @@
 module PmdTester
   class Cmd
 
-    PMD_EXIT_STATUS = 4
-
     def self.execute(cmd)
       puts cmd
 
       stdin, stdout, stderr, wait_thr = Open3.popen3("#{cmd};")
```

Comment:
I'm having some trouble with running the test TestPmdReportBuilder locally. It seems, that the processes block (e.g. maven or pmd itself) until the output stream/err stream is consumed before they exit.
This is maybe the reason, why this PR doesn't build on travis - there is no output for more than 10 minutes, and then the build is automatically aborted.

I've seen this bug report: https://bugs.ruby-lang.org/issues/9082
At the very end, they suggest to use `capture3` instead of `popen3`, which seems to be an easy solution.
Note: It has quite a different syntax (e.g. doesn't return the stdin pipe, returns directly the exit value...)